### DISCUSSION TITLE
Is this project still ongoing?

### DISCUSSION THREAD

**Discussion by Zaya6** [dfranx/SHADERed#310] [General]
This tool has been a great help to me for a while now and I'm curious to see if the project is still alive. I work in a large company that has a legacy engine and codebase and still uses hlsl shaders. This tool has been so instrumental in developing new and updated shaders for our products. I'm not in leadership or anything, just a tech artist in one of the studios. 

When I look at the issues I see many going unanswered for more than a year, does that mean this project is dead?

**Comment by jto-daA** [comment]
I wasn't getting emails for new Issues, but definitely appreciate this project. I haven't used it in a while  because I have done any Shader code, but will soon. I will slowly work through these issues just because I think the project is awesome!
